### ISSUE TITLE
Support SARIF as Report format

### ISSUE THREAD

**Issue by Jeeppler** [find-sec-bugs/find-sec-bugs#637]
## Description

Support the [Static Analysis Results Interchange Format (SARIF)](https://docs.oasis-open.org/sarif/sarif/v2.1.0/sarif-v2.1.0.html) as report format. SARIF is a JSON based and can be used to interchange reports between tools supporting SARIF.

SARIF is:

- based on JSON
- a report format for static analysis tools, like FindSecBugs
- standardized format
- can be used to exchange information

Standard: https://docs.oasis-open.org/sarif/sarif/v2.1.0/sarif-v2.1.0.html
Tutorials: https://github.com/microsoft/sarif-tutorials
Validator: https://sarifweb.azurewebsites.net/Validation

**Comment by Jeeppler** [comment]
It is already supported. Sorry.

**Comment by leandro-lorenzini** [comment]
> It is already supported. Sorry.

Is it already supported? I can't see it in the [documentation](https://github.com/find-sec-bugs/find-sec-bugs/wiki/CLI-Tutorial#additional-arguments).

**Comment by Jeeppler** [comment]
@leandro-lorenzini yes, it does. Simply add: `-sarif`. For example: `findsecbugs-cli-1.11.0/findsecbugs.sh -progress -sarif -output report.sarif.json vulnerable-web-server/build/classes/`
